portafly/tsconfig.json

```
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src",
```

This should point to the root dir, not the src folder. Is there any reason why? In my IDE it won't work unless:
```suggestion
    "baseUrl": ".",
```

---

This along with the custom paths made it possible for using them, we should revise your IDE config maybe?